### DISCUSSION TITLE
will the AccordionState PR be merged?

### DISCUSSION THREAD

**Discussion by weevolve-dev** [adobe/react-spectrum#3131] [Q&A] [answered]
Hi everyone,

I saw that there is a pending [pull request](https://github.com/adobe/react-spectrum/pull/3033), which seems to address the issue that accordions can currently [not have JSX items](https://github.com/adobe/react-spectrum/issues/1989) as content.

Is there any chance this PR will be merged and make it into the next release?

Thanks!

**Comment by dannify** [comment, marked as answer]
Hey @weevolve-dev, 
The PR mentioned was made by a contributor however, at the moment the core team is concentrating on a few other components and we do not have the resources to pick up it up. We will look at moving that component along later in the year most likely.
